### ISSUE TITLE
`no-console` and `no-window` warn on object property names

### ISSUE THREAD

**Issue by 0f-0b** [denoland/deno_lint#1232]
### Lint Name

`no-console`, `no-window`

### Code Snippet

```ts
interface Globals { console: any }
const globals: Globals = { console };
const params: { window: number } = { window: 128 };
globals.console.log(params.window);
```

### Expected Result

```shellsession
$ dlint run --config .dlint.json a.ts
no-console

  × `console` usage is not allowed.
   ╭─[a.ts:2:28]
 2 │ interface Globals { console: any }
 3 │ const globals: Globals = { console };
   ·                            ───────
 4 │ const params: { window: number } = { window: 128 };
   ╰────
  help: https://lint.deno.land/#no-console

Found 1 problem
```

### Actual Result

```shellsession
$ dlint run --config .dlint.json a.ts
no-console

  × `console` usage is not allowed.
   ╭─[a.ts:1:21]
 1 │ interface Globals { console: any }
   ·                     ───────
 2 │ const globals: Globals = { console };
   ╰────
  help: https://lint.deno.land/#no-console

no-console

  × `console` usage is not allowed.
   ╭─[a.ts:2:28]
 2 │ interface Globals { console: any }
 3 │ const globals: Globals = { console };
   ·                            ───────
 4 │ const params: { window: number } = { window: 128 };
   ╰────
  help: https://lint.deno.land/#no-console

no-window

  × window is deprecated and scheduled for removal in Deno 2.0
   ╭─[a.ts:3:17]
 3 │ const globals: Globals = { console };
 4 │ const params: { window: number } = { window: 128 };
   ·                 ───┬──
   ·                    ╰── Instead, use `globalThis`
 5 │ globals.console.log(params.window);
   ╰────
  help: https://lint.deno.land/#no-window

no-window

  × window is deprecated and scheduled for removal in Deno 2.0
   ╭─[a.ts:3:38]
 3 │ const globals: Globals = { console };
 4 │ const params: { window: number } = { window: 128 };
   ·                                      ───┬──
   ·                                         ╰── Instead, use `globalThis`
 5 │ globals.console.log(params.window);
   ╰────
  help: https://lint.deno.land/#no-window

no-console

  × `console` usage is not allowed.
   ╭─[a.ts:4:9]
 4 │ const params: { window: number } = { window: 128 };
 5 │ globals.console.log(params.window);
   ·         ───────
   ╰────
  help: https://lint.deno.land/#no-console

no-window

  × window is deprecated and scheduled for removal in Deno 2.0
   ╭─[a.ts:4:28]
 4 │ const params: { window: number } = { window: 128 };
 5 │ globals.console.log(params.window);
   ·                            ───┬──
   ·                               ╰── Instead, use `globalThis`
   ╰────
  help: https://lint.deno.land/#no-window

Found 6 problems
```

### Additional Info

```shellsession
$ cat .dlint.json
{
  "rules": {
    "include": [
      "no-console",
      "no-window"
    ]
  }
}
```

### Version

```shellsession
$ dlint --version
dlint 0.54.0
```
